tests/test_unit/test_sample_data.py

```diff
@@ -44,7 +44,7 @@ def validate_metadata(metadata: dict[str, dict]) -> None:
     metadata_fields = [
         "sha256sum",
         "type",
-        "source_software",
+        "source_format",
```

Comment:
Needs to be changed back to `source_software` to make the tests pass (until we update the corresponding field in the sample data repository).
```suggestion
        "source_software",
```